js/danoni_main.js

```
@@ -549,46 +549,102 @@ const formatObject = (_obj, _indent = 0, { colorFmt = true, rootKey = `` } = {})
+	};
+
+	/**
+	 * 配列・オブジェクトのネスト整形処理
+	 * @param {any} _correction 
+	 * @param {number} _indent 
+	 * @param {boolean} colorFmt
+	 * @param {string} rootKey 
+	 * @returns {string}
+	 */
+	const formatCollection = (_correction, _indent, { colorFmt, rootKey }) => {
+		const baseIndent = getIndent(_indent);
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)